pkg/apiserver/registry/system/supportbundle/rest.go

```diff
@@ -308,6 +308,10 @@ func (r *supportBundleREST) clean(ctx context.Context, bundlePath string, durati
 	defaultFS.Remove(bundlePath)
 }
 
+func (r *supportBundleREST) GetSingularName() string {
+	return "supportBundle"
```

Comment:
Unless I am mistaken, this should be `"supportbundle"`, not `"supportBundle"`

Reply:
Fixed.